stable/artifactory-ha/values.yaml

```diff
@@ -1209,6 +1212,12 @@ artifactory:
       type: ""
       topologyKey: "kubernetes.io/hostname"
 
+frontend:
+  ## Session settings
+  session:
+    ## After how much time the frontend token need to be refreshed
```

Comment:
`Time in minutes after which the frontend token will need to be refreshed`

Reply:
Done